public/translations/pt-BR.json

```diff
@@ -477,5 +479,18 @@
       "primary_approve": "sim, aprovar",
       "secondary": "não, cancelar"
     }
+  },
+  "tranferscreen": {
```

Comment:
please use something different, here is a few suggestions:

- transfer
- transfer_result
- transfer_history


etc